bindings/node/README.md

```diff
@@ -621,6 +677,29 @@ the original ones.
 ## MongoCryptError
 An error indicating that something went wrong specifically with MongoDB Client Encryption
 
+<a name="loadCredentials"></a>
+
+## loadCredentials(kmsProviders)
```

Comment:
Similarly here, this should probably not be included in the README. If it is, it should also be tagged as internal, like above